spk/tvheadend/src/service-setup.sh

```diff
@@ -2,7 +2,12 @@
 # Sourced script by generic installer and start-stop-status scripts
 
 # Add ffmpeg and ifself to path
-PATH="${SYNOPKG_PKGDEST}/bin:/var/packages/ffmpeg/target/bin:${PATH}"
+PYTHON_DIR="/var/packages/python38/target"
+PYTHONENV="${SYNOPKG_PKGDEST}/env"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+WHEELHOUSE=${SYNOPKG_PKGDEST}/share/wheelhouse
+FFMPEG_DIR="/usr/local/ffmpeg"
```

Comment:
`/usr/local/{package}` folders must be avoid for DSM7 compatibility.

```suggestion
FFMPEG_DIR="/var/packages/ffmpeg/target"
```